### ISSUE TITLE
mitochondrial, chloro, and other plastid genomes

### ISSUE THREAD

**Issue by AntonioBaeza** [ythuang0522/homopolish#52]
I wonder if you are interested in developing sketches for mitochondrial or plastic genomes?

**Comment by ythuang0522** [comment]
No. You may consider using local library (`-l`) by pulling closely-related sequences as you wish.
`python3 homopolish.py polish -a yourgenome.fasta -l path_to_your_genomes.fasta -m R9.4.pkl -o youroutput`

**Comment by AntonioBaeza** [comment]
thank you, I will give it a try
